conf/logs.json

```diff
@@ -136,5 +136,52 @@
     "hints": {
       "records": "Records[*]"
     }
+  },
+  "cloudwatch:cloudtrail": {
```

Comment:
@ryandeivert and I did a lot of testing regarding this.  We came to the following conclusion: Cloudtrail logs sent via Cloudwatch Events will always have the schema as defined here.  If a user is filtering on a specific API request (like EC2 for instance), they will need to define a specific schema.